tsptw_data_dt.h

```diff
@@ -752,6 +756,11 @@ void TSPTWDataDT::LoadInstance(const std::string& filename) {
     }
   }
 
+  // compute earliest start first
+  for (const ortools_vrp::Vehicle& vehicle : problem.vehicles()) {
+    earliest_start_ = std::min((int64)vehicle.time_window().start(), earliest_start_);
```

Comment:
Does this work if the vehicle doesn't have any time windows -- i.e., does it set `earliest_start_` to 0 as expected?

Reply:
In protobuf the default value for a field not provided is 0. Moreover, ortools.rb systematically provide the value 0 if the vehicle timewindow start does not exist

Reply:
My question was "What if the vehicle has no time_window" then what does  `vehicle.time_window()` return? 0, empty `time_window` object, something else?

Basically, did you try it with an instance which has vehicles with and without time windows?

Reply:
From the perspective of ortools.rb a vehicle always have a timewindow even when there is no timewindow provided in the VRP